src/icrar/leap-accelerate/tests/algorithm/PhaseRotateTests.cc

```diff
@@ -114,7 +114,9 @@ namespace icrar
             }
             else if(impl == ComputeImplementation::cuda)
             {
+#ifdef CUDA_ENABLED
```

Comment:
Same comment regarding rutime v/s compile time check, same for the rest of the file.

Reply:
here the cuda vector functions do not exist if CUDA_ENABLED  is not defined.

Reply:
Same comment as above: you are right, I hadn't spotted that there was CUDA-specific code invoked within the `ifdef`, but on the other hand now that you have `icrar::Calibrate` you can probably reuse it here.